tests/test_image_caption.py

```diff
@@ -0,0 +1,173 @@
+"""Unit tests for the `image_caption_dataset` module."""
+
+import os
+
+import pytest
+
+from nordjylland_news.image_caption_dataset import ImageCaptionDataSetBuilder
+
+PAGE = 1
```

Comment:
Det her burde nok være en fixture i stedet:
```suggestion
@pytest.fixture(scope="module")
def page() -> Generator[int, None, None]:
    yield 1
```

Og så inkludere `page` som argument i funktionerne, der bruger det.